osquery/tables/system/windows/drivers.cpp

```diff
@@ -264,20 +260,14 @@ QueryData genDrivers(QueryContext& context) {
     }
 
     if (r.count("service") > 0 && !r.at("service").empty()) {
-      auto svc_key = kServiceKeyPath + r["service"];
-      r["service_key"] = svc_key;
-
-      // If the image map doesn't exist in the cache, manually look it up
-      auto svc_key_it = svc_image_map.find(svc_key);
-      if (svc_key_it != svc_image_map.end()) {
-        r["image"] = svc_key_it->second;
-      } else {
-        // Manual lookups of the service keys image path are _very_ slow
-        VLOG(1) << r["service"]
-                << " not found in image cache, performing manual lookup";
-        r["image"] = getDriverImagePath(svc_key);
+      std::string path;
+      r["service_key"] = kServiceKeyPath + r["service"];
+      auto ret = getDriverImagePath(r["service"], path);
+      if (ret.ok()) {
```

Comment:
Here we never print the error message stored in the `Status` return value. Can we print the message + error code in `VLOG(1)`?

Although you haven't touched this part of the code here https://github.com/osquery/osquery/pull/7444/files#diff-ccecde3b2470ca06721a35a78839a429e0fce6cc958f9365d9d7e877fda94262R254, could you also check the return value before accessing `val` and also log with `VLOG(1)` if it's an error, since it's currently totally ignored?

Reply:
Thanks for comment. I've added the missing logging.